docs/reference/query-dsl/bool-query.asciidoc

```diff
@@ -81,7 +81,7 @@ all documents where the `status` field contains the term `active`.
 This first query assigns a score of `0` to all documents, as no scoring
 query has been specified:
 
-[source,json]
+[source,js]
```

Comment:
No idea really, though in general I'm in favor of consistency - makes it easier for new comers to adopt existing style.